components/collective-page/BudgetStats.js

```diff
@@ -103,19 +103,21 @@ const BudgetStats = ({ collective, stats, horizontal }) => {
                   textTransform="uppercase"
                   color="black.700"
                   extraTooltipContent={
-                    <Fragment>
-                      <Box mt={2}>
-                        <FormattedMessage
-                          id="budgetSection-balance-consolidated"
-                          defaultMessage="Total consolidated including Projects and Events: {amount}"
-                          values={{
-                            amount: formatCurrency(stats?.consolidatedBalance.valueInCents || 0, collective.currency, {
-                              locale,
-                            }),
-                          }}
-                        />
-                      </Box>
-                    </Fragment>
+                    stats.consolidatedBalance?.valueInCents && (
+                      <Fragment>
+                        <Box mt={2}>
+                          <FormattedMessage
+                            id="budgetSection-balance-consolidated"
+                            defaultMessage="Total consolidated including Projects and Events: {amount}"
+                            values={{
+                              amount: formatCurrency(stats.consolidatedBalance.valueInCents || 0, collective.currency, {
+                                locale,
+                              }),
+                            }}
+                          />
+                        </Box>
+                      </Fragment>
+                    )
```

Comment:
Make the "consolidated balance" section optional in case it's not returned (for whatever reason).